### PR TITLE
Clarify build process & repo documentation

### PR DESCRIPTION
**Pull request by sknep**
Please check this thoroughly, as I've changed the build tasks considerably. Netlify configs for this repo may also need to be adjusted.

I saw that the sass wasn't compiling in the dev and build tasks, and some other opportunities for clarification (like the Readme content, which makes it sound like _this_ is the repo for the actual Astro UXDS).

**Please pull, install, build, etc, to test this PR.**  
Live build is here: https://deploy-preview-67--astrouxds.netlify.app/ but any changes should be imperceptible from Master.

This should get merged into Draft as well, as soon as approved, to keep build processes consistent.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,20 +9,23 @@
 	"main": "index.js",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"sass": "node-sass -rw scss -o css",
-		"searchIndex": "node ./_includes/search_config/search-generator.js",
-		"dev": "eleventy --serve && npm run sass",
```
Comment (sknep) [inline package.json:14]: when running this task, eleventy would work but we'd never get to the sass task

File: package.json [hunk 2]
```diff
@@ -9,20 +9,23 @@
...
-		"prebuild": "npm run searchIndex",
-		"build": "eleventy"
+		"start": "cross-env run-p -l start:*",
+		"start:search": "node ./_includes/search_config/search-generator.js",
+		"start:scss": "node-sass -rw scss -o css",
+		"start:eleventy": "eleventy --serve",
+		"build": "cross-env run-s -l build:*",
+		"build:clean": "rimraf _site",
+		"build:search": "node ./_includes/search_config/search-generator.js",
+		"build:scss": "node-sass -r scss -o css",
```
Comment (sknep) [inline package.json:19]: @im-cr Do you think the output should be minified or not? You dealt with this recently, it seems like

File: package.json [hunk 3]
```diff
@@ -9,20 +9,23 @@
 	"main": "index.js",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"sass": "node-sass -rw scss -o css",
-		"searchIndex": "node ./_includes/search_config/search-generator.js",
-		"dev": "eleventy --serve && npm run sass",
-		"preserve": "npm run searchIndex",
-		"serve": "eleventy --serve",
-		"prebuild": "npm run searchIndex",
-		"build": "eleventy"
```
Comment (sknep) [inline package.json:18]: As you can see here, build never attempted to run the sass task

File: css/chapters.css [hunk 4]
```diff
@@ -16,30 +15,28 @@ main {
 /* main :not(figure):not(img):not(table) {
 	max-width: var(--content-max-text-width, 40em);
 } */
- main ol:not(.compliance-list) {
```
Comment (sknep) [inline css/chapters.css:19]: Note: this is just a whitespace change now that the css is actually being built from the sass, instead of updated by hand.

**Comment by sknep** [comment]
@dmcalester @im-cr can I have your reviews on this PR?

File: package.json [hunk 5]
```diff
@@ -32,11 +34,12 @@
 		"markdown-it-div": "^1.1.0",
 		"markdown-it-implicit-figures": "^0.9.0",
 		"markdown-json": "^1.4.1",
-		"marked": "^0.8.0"
```
Comment (sknep) [inline package.json:35]: from what i can tell, we aren't using this markdown parser https://www.npmjs.com/package/marked

File: package.json (hunk 2 above)
Comment (dmcalester) [reply package.json:19]: Yes, output should be minified.

File: scss/chapters.scss [hunk 6]
```diff
@@ -25,16 +25,13 @@ main {
 	max-width: var(--content-max-text-width, 40em);
 } */
 
-body:not(.compliance) {
```
[the remark this answers is not in the document]
Comment (dmcalester) [reply scss/chapters.scss:28]: @im-cr please ensure this style change is accurately reflected on `draft`

File: package.json (hunk 2 above)
Comment (sknep) [reply package.json:19]: @dmcalester does that mean checking in the minified CSS? I think it's checked in now. If it's minified in build but not on serve, then whichever the committer ran last will affect the files on the branch.

**Comment by sknep** [comment]
thanks duncan! did you make sure the netlify tasks matched the changes to build process?

**Pull request merged**